ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml

```
@@ -62,6 +62,13 @@ test_binary_build_commands: |
   go mod vendor
   mkdir /.cache && chmod 775 -R /.cache ${GOPATH}
 tests:
+- as: publish-multi-arch-images
+  postsubmit: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      DOCKERFILE_IMAGE_PAIRS: Dockerfile.assisted-installer,quay.io/edge-infrastructure/assisted-installer-multi:latest|Dockerfile.assisted-installer-controller,quay.io/edge-infrastructure/assisted-installer-controller-multi:latest
```

```suggestion
      DOCKERFILE_IMAGE_PAIRS: >-
        Dockerfile.assisted-installer,quay.io/edge-infrastructure/assisted-installer-multi:latest |
        Dockerfile.assisted-installer-controller,quay.io/edge-infrastructure/assisted-installer-controller-multi:latest
```
assuming we can tolerate the little space we'll get there (obviously the one before the '|' but also the other one after that '|')